contrib/bsp/src/mill/contrib/bsp/MillBuildServer.scala

```diff
@@ -132,24 +132,28 @@ class MillBuildServer(evaluator: Evaluator, bspVersion: String, serverVersion: S
       val millBuildTargetId = getMillBuildTargetId(evaluator)
 
       val items = dependencySourcesParams.getTargets.asScala
-        .filter(_ != millBuildTargetId)
```

Comment:
We used to not resolve source dependencies for the Mill build dependencies. Which made code navigation hard since all we could see were the signature only.